inferno/callbacks.py

```diff
@@ -163,7 +164,8 @@ def _scoring(self, net, X, y):
 
         return score
 
-    def on_batch_end(self, net, X, y, train, *args, **kwargs):
+    # pylint: disable=unused-argument,arguments-differ
```

Comment:
The parent's class sig is just `on_batch_begin(self, net, **kwargs)`. We could extend that to match, not sure if we want to go that route.

The `**kwargs` for child classes should be enforced in case we ever decide to pass more arguments to the method. The `*args` might be excessive, should I remove it?

Reply:
Nope is OK